package.json

```diff
@@ -14,16 +14,40 @@
     "test": "echo \"Write tests! -> https://gatsby.dev/unit-testing\" && exit 1"
   },
   "dependencies": {
-    "gatsby": "^2.24.50",
+    "@material-ui/core": "^4.11.0",
+    "acorn": "^6.4.1",
+    "babel-plugin-styled-components": "^1.11.1",
+    "babel-upgrade": "^1.0.1",
+    "eslint": "^7.14.0",
+    "eslint-config-prettier": "^6.15.0",
+    "gatsby": "^2.24.65",
+    "gatsby-background-image": "^1.1.2",
+    "gatsby-cli": "^2.12.98",
+    "gatsby-graphql-source-toolkit": "^0.5.0",
+    "gatsby-image": "^2.4.19",
+    "gatsby-plugin-material-ui": "^2.1.10",
+    "gatsby-plugin-sass": "^2.3.13",
+    "gatsby-plugin-scroll-reveal": "0.0.7",
+    "gatsby-plugin-sharp": "^2.6.36",
+    "gatsby-plugin-smoothscroll": "^1.2.0",
+    "gatsby-plugin-styled-components": "^3.3.11",
+    "gatsby-source-filesystem": "^2.3.30",
+    "gatsby-transformer-sharp": "^2.5.15",
+    "node-sass": "^4.14.1",
     "react": "^16.12.0",
-    "react-dom": "^16.12.0"
+    "react-dom": "^16.12.0",
+    "react-intersection-observer": "^8.31.0",
+    "react-markdown": "^4.3.1",
+    "react-refresh": "^0.8.3",
+    "styled-components": "^5.2.0"
```

Comment:
It seems like `react-icons` is missing. Without it I can't start the project. I installed it on my side and it works now